app/config/variables.php

```diff
@@ -224,7 +224,7 @@
                 'filter' => ''
             ],
             [
-                'name' => '_APP_USAGE_AGGREGATION_INTERVAL',
+                'name' => '_APP_STATS_AGGREGATION_INTERVAL',
```

Comment:
We cant just rename the variable.. We need to introduce a new variable `_APP_STATS_AGGREGATION_INTERVAL` and mark `_APP_USAGE_AGGREGATION_INTERVAL` as deprecated.

Reply:
Reverted the ENV change in this PR, will create a separate PR to handle it.